packages/core/stories/button.doc.stories.mdx

```diff
@@ -100,7 +62,9 @@ import { Button } from "@jpmorganchase/uitk-core";
 
 ## Props
 
-<ArgsTable of={Button} />
+<Story name="Props">
```

Comment:
I understand the benefits but I still don't think we should declare stories in mdx.

Reply:
To avoid stories showing up in the sidebar.